### ISSUE TITLE
Requirements pylibmc fails to install

### ISSUE THREAD

**Issue by Argetan** [twschiller/open-synthesis#175]
I tried installing the requirements but it threw the following error:
```
Command "/Users/usernameIntentionallyLeftOut/Python/venvs/open-synthesis/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/pl/_ygj9sgj2t3ggn6j66qb7wsm0000gn/T/pip-install-rvx1eod4/pylibmc/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/pl/_ygj9sgj2t3ggn6j66qb7wsm0000gn/T/pip-record-2j2eoo8p/install-record.txt --single-version-externally-managed --compile --install-headers /Users/julian/Python/venvs/open-synthesis/include/site/python3.6/pylibmc" failed with error code 1 in /private/var/folders/pl/_ygj9sgj2t3ggn6j66qb7wsm0000gn/T/pip-install-rvx1eod4/pylibmc/
```

I found the issue, being that I didn't have `libmemcached` installed on my system (OS X). It can be fixed by installing it using HomeBrew: `brew install libmemcached`.

I created this issue to inform other users, feel free to close it if you don't think it's necessary.

**Comment by twschiller** [comment]
Thanks @Argetan. Will add to the project setup instructions

**Comment by twschiller** [comment]
Fixed in faa7c786009e7220c6ad59dc2465903b41259f07
